SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Leaflet takes security vulnerabilities very seriously and appreciates your responsible disclosure efforts. We will make every effort to acknowledge your contributions and handle them promptly.
+To report a security issue, please send your report to agafonkin@gmail.com. We recommend encrypting the email if possible.
```

Comment:
Let's not complicate things, users that are aware of PGP are sure to use it either way.
```suggestion
To report a security issue, please send your report to agafonkin@gmail.com.
```